CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Fixes
 
+- **GCP Stackdriver Scalar:** Update Stackdriver client to handle detecting double and int64 value types ([#3777](https://github.com/kedacore/keda/issues/3777))
```

Comment:
please, move this to the last line in the section, "**General..." is the first block, and then we sort the others alphabetically

Reply:
The log entry has been moved below General and into the list alphabetically.